### ISSUE TITLE
recurring tasks

### ISSUE THREAD

**Issue by jvd88** [projectfork/Projectfork#1526]
Hi,

it is mentioned earlier, but recurring tasks should be really implemented.
we have a few tasks that occur every day. It is a pain in the a.. to make that task every day.

Is there already something or is it planned ?

joep

**Comment by angek** [comment]
I agree. it is something that should be considered. I think right now we need to focus on getting existing bugs sorted and get things back on track.

**Comment by doriancin** [comment]
I agree too!! This will be great for weekly tasks of for testing!
